### ISSUE TITLE
[JOSS Review] DOIs in references

### ISSUE THREAD

**Issue by nhejazi** [mikldk/disclapmix#1]
As per a requirement of the [ongoing review of this R package for JOSS](https://github.com/openjournals/joss-reviews/issues/748), a few minor additions are necessary in the file `paper.bib`, and any place else in the R package in which bibliographic information is stored. Primarily, DOIs should be added to each article that has been assigned one. For example, the first entry in `paper.bib` should look like the following, where the last two lines are new.

```
@article{AndersenDisclap2013,
	 Author = {Mikkel Meyer Andersen and Poul Svante Eriksen and Niels Morling},
	 Journal = {Journal of Theoretical Biology},
	 Pages = {39--51},
	 Title = {{The discrete Laplace exponential family and estimation of Y-STR haplotype frequencies}},
	 Volume = {329},
         Year = {2013},
         doi = {10.1016/j.jtbi.2013.03.009},
         url = {https://doi.org/10.1016/j.jtbi.2013.03.009}
}
```

**Comment by mikldk** [comment]
Sorry about this. Fixed in 5d3db765054c5b9850be4809874405a378b714f4 (I couldn't find DOI for some of the older references). Will you confirm before I close this?

**Comment by nhejazi** [comment]
👍 Thanks for making the change. LGTM --- feel free to close.
